### PR TITLE
Remove no-op return statements.

### PR DESCRIPTION
**Pull request by SpacePossum**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| License | MIT |

For ref:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1591

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Tests/Controller/ProfilerControllerTest.php [hunk 1]
```diff
@@ -63,7 +63,6 @@ public function testReturns404onTokenNotFound()
                     return new Profile($token);
                 }
```
Comment (keradus) [inline src/Symfony/Bundle/WebProfilerBundle/Tests/Controller/ProfilerControllerTest.php]: this line is not needed

File: src/Symfony/Component/Intl/Data/Generator/CurrencyDataGenerator.php [hunk 2]
```diff
@@ -122,7 +122,6 @@ protected function generateDataForLocale(BundleReaderInterface $reader, $tempDir
             return $data;
         }
```
Comment (keradus) [inline src/Symfony/Component/Intl/Data/Generator/CurrencyDataGenerator.php]: same here

File: src/Symfony/Bundle/WebProfilerBundle/Tests/Controller/ProfilerControllerTest.php (hunk 1 above)
Comment (keradus) [inline src/Symfony/Bundle/WebProfilerBundle/Tests/Controller/ProfilerControllerTest.php]: here and in every other places

Comment (SpacePossum) [inline src/Symfony/Bundle/WebProfilerBundle/Tests/Controller/ProfilerControllerTest.php]: :+1: thanks,  done and squashed into one clean-up commit.

File: src/Symfony/Component/Finder/Adapter/AbstractFindAdapter.php [hunk 3]
```diff
@@ -94,7 +94,7 @@ public function searchInDirectory($dir)
         $command->setErrorHandler(
             $this->ignoreUnreadableDirs
                 // If directory is unreadable and finder is set to ignore it, `stderr` is ignored.
-                ? function ($stderr) { return; }
+                ? function ($stderr) {  }
```
Comment (xabbuh) [inline src/Symfony/Component/Finder/Adapter/AbstractFindAdapter.php]: There should be only one space between the braces.

Comment (xabbuh) [inline src/Symfony/Component/Finder/Adapter/AbstractFindAdapter.php]: same below

Comment (SpacePossum) [inline src/Symfony/Component/Finder/Adapter/AbstractFindAdapter.php]: thank you, fixed :)

**Comment by xabbuh** [comment]
:+1: 

Status: Reviewed

**Comment by stof** [comment]
I suggest waiting for the decision on https://github.com/symfony/symfony/issues/17201 to do this, as these functions are meant to return null, not void.

**Comment by SpacePossum** [comment]
@stof good point, good to see symfony will start to handle `void` and `null` as diff. values :)

**Comment by SpacePossum** [comment]
ping me when the fixer can be moved to SF level ;)

**Pull request closed without merging**